ratatool-shapeless/src/main/scala/com/spotify/ratatool/shapeless/CaseClassDiffy.scala

```diff
@@ -43,7 +43,7 @@ object MapEncoder {
   MapEncoder[FieldType[K, String]] = {
     val name = witness.value.name
     createEncoder { v =>
-      Map(name -> v.self)
+      Map(name -> v.toString)
```

Comment:
What was the issue here?

Reply:
In 212 this is a `WrappedString` (https://github.com/scala/scala/blob/v2.12.10/src/library/scala/collection/immutable/WrappedString.scala#L34) 

In 2.13 it is just a `String` so `.self` won't compile.

`toString` in 2.12 returns self anyways so this is equivalent in 2.12 and doesn't break in 2.13.